### PR TITLE
create JLKwong.md

### PR DESCRIPTION
**Pull request by JLKwong**
Fixes #issue_number

### Description
Description of changes made.

### Raw.Githack preview link
https://raw.githack.com/JLKwong/JLKwong.github.io/fork-tutorial-step-3/pages/vi/profiles/JLKwong.md

### CODE REVIEW

File: pages/vi/profiles/JLKwong.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* **Upgraded 80+ computers for 3 companies** from Windows 7 to Windows 10 using ConnectWise Control (ScreenConnect) and Windows Deployment Services at an IT internship with Brydan Solutions  
+* **Restored an empty 100-cubicle home office building back to full operations within 2 weeks** after a debilitating malware attack at Marshall Retail Group  
+* **Quarantined and restored 100+ PC's from scratch** after a devastating Mount Locker ransomware attack by identifying infected PC's, disconnecting them from the network, powering them off, imaging them, and reinstalling software according to each department's needs  
+* **Communicated hourly and daily updates with the IT team** during the Mount Locker ransomware attack to triage and restore the company home office back to full operations within 2 weeks
+
+I am a resilient, talented, and high-potential prospective software engineer intern at the Open Learning Exchange. By the end of 2021, I hope to kickstart my career in software engineering. :D Add me on LinkedIn: <a>https://www.linkedin.com/in/jamielk/</a>
```
Comment (dogi) [inline pages/vi/profiles/JLKwong.md]: @JLKwong please use markdown and not html

Comment (JLKwong) [reply pages/vi/profiles/JLKwong.md]: Oop, my bad. I'll fix that in a bit. xD

Comment (JLKwong) [reply pages/vi/profiles/JLKwong.md]: I used a couple Markdown stuffs. It should be good now. ^-^

**Comment by rjpadilla** [comment]
You may want to change `https://raw.githack.com/JLKwong/JLKwong.github.io/fork-tutorial-step-3/pages/vi/profiles/JLKwong.md` to `https://raw.githack.com/JLKwong/JLKwong.github.io/fork-tutorial-step-3/#!pages/vi/profiles/JLKwong.md` in the description.
    Appending the hashbang `#!` before `pages/vi` directs you to the live production of your profile page.
    Leaving it out pops-up a save window instead. Other than that you have a great looking resume! 💯

**Review by dogi — APPROVED** [review]

**Pull request merged**